lib/releases.sh

```diff
@@ -65,56 +90,60 @@ function get_latest_release() {
   fi
 }
 
-# CAPM3, CAPI and BMO release path
-CAPIRELEASEPATH="{https://api.github.com/repos/${CAPI_BASE_URL:-kubernetes-sigs/cluster-api}/releases}"
-
+CAPIGOPROXY="https://proxy.golang.org/sigs.k8s.io/cluster-api/@v/list"
 # CAPM3, CAPI and BMO releases
-if [ "${CAPM3RELEASEBRANCH}" = "release-1.6" ]; then
+if [[ "${CAPM3RELEASEBRANCH}" = "release-1.6" ]]; then
   # 1.6.99 points to the head of the release-1.6 branch. Local override for CAPM3 is created for this version.
   export CAPM3RELEASE="v1.6.99"
-  export CAPIRELEASE="${CAPIRELEASE:-$(get_latest_release "${CAPIRELEASEPATH}" "v1.6.")}"
```

Comment:
nit: would be great to avoid duplication by setting e.g. CAPI_RELEASE_PREFIX="v1.6." and then only setting CAPIRELEASE once after the "if" block